test/integration/wdioUtils.js

```diff
@@ -17,14 +36,89 @@ async function setSessionCookie () {
     }]);
 }
 
-async function login() {
-    await browser.url(makeURL('narrative/static/kbase/config/config.json'));
-    await setSessionCookie();
-    return;
+/**
+* "Types" a string into the wdio browser.
+* Webdriver simulates user keyboard input by individual key presses.
+* This function makes providing user keyboard easier by abstracting 
+* input of a string.
+* Note that the keys are sent to the currently focused control in the browser.
+* @arg {string} stringToSend - the string to "type" into the browser
+* @returns {Promise} Promise value is ignored
+*/
+async function sendString(stringToSend) {
+    for (const letter of stringToSend.split()) {
+        await browser.keys(letter);
+    }
+}
+
+async function clickWhenReady(el) {
```

Comment:
Thanks. You have no idea! Or maybe you do. 
Especially in the narrative where the covering overlay blocks the rest of the view while the ui is being built, and elements will be available, but not yet visible or interactable.
And with the generally dynamic nature of the narrative, where things pop into place after some time passes, and this can happen in layers, this kind of wrapping of a test or action with a "wait" style check, whether canned in a function or just a programming pattern, is common.